cmake/configs/nuttx_aerocore2_default.cmake

```diff
@@ -150,4 +151,4 @@ set(config_module_list
 
 	# Hardware test
 	#examples/hwtest
-)
+)
```

Comment:
You somehow killed all empty lines at the end of the files which should be there by definition (see the red sign by GitHub).

Reply:
It's whatever git check is catching and complaining about. See git `core.whitespace`.